pkg/beacon/relay/entry/states.go

```diff
@@ -136,12 +142,31 @@ func (scs *signatureCompleteState) Initiate(ctx context.Context) error {
 				err,
 			)
 		} else {
-			logger.Debugf(
-				"[member:%v] accepting signature share from member [%v]",
-				scs.MemberIndex(),
-				message.senderID,
-			)
-			seenShares[message.senderID] = share
+			publicKey, ok := scs.signer.GroupPublicKeyShares()[message.senderID]
+			if !ok {
+				logger.Warningf(
+					"[member:%v] could not validate signature share from "+
+						"member [%v] because their public key was not found",
```

Comment:
`because their public key was not found` or `because their public key share was not found`?

We can also shorten it a little:
```
[member:%v] could not validate signature share from member [%v]; public key share not found
```